src/components/product-parameter/ProductParameter.jsx

```diff
@@ -0,0 +1,49 @@
+import React, { useState } from 'react'
+import PropTypes from 'prop-types'
+import LabelledInput from '../labelled-input'
+
+const ProductParameter = ({ text, options, onChange, ...other }) => {
+  const [defaultValue, setDefaultValue] = useState('Выбрать')
+  const [option, setOption] = useState({ name: 'my name', value: 'my value' })
+
+  const selectInput = (
+    <select
+      name={text}
+      onChange={() => {
+        setDefaultValue(null)
+        setOption({ value: 'Hi', name: 'Bye' })
+      }}
+    >
+      {defaultValue ? (
```

Comment:
It's wonderful solution to decide whether render defaultOption or not this way! 

I thought about somehow modifying options array and add default option there if required, but this solution is much more cleaner, simple and stable! 👍